src/state/slices/config_slice.ts

```diff
@@ -0,0 +1,67 @@
+import { Color } from "three";
+import { StateCreator } from "zustand";
+
+import {
+  DrawMode,
+  DrawSettings,
+  OUT_OF_RANGE_COLOR_DEFAULT,
+  OUTLIER_COLOR_DEFAULT,
+  OUTLINE_COLOR_DEFAULT,
+  TabType,
+} from "../../colorizer";
+
+type ConfigSliceState = {
+  showTrackPath: boolean;
+  showScaleBar: boolean;
+  showTimestamp: boolean;
+  showLegendDuringExport: boolean;
+  showHeaderDuringExport: boolean;
+  outOfRangeDrawSettings: DrawSettings;
+  outlierDrawSettings: DrawSettings;
+  outlineColor: Color;
+  openTab: TabType;
```

Comment:
This might eventually need to live in a different slice, especially if we ever revamp the layout!